suzieq/cli/sqcmds/FsCmd.py

```diff
@@ -0,0 +1,57 @@
+import time
+from nubia import command, argument
+
+from suzieq.cli.sqcmds.command import SqCommand
+from suzieq.sqobjects.fs import FsObj
+
+
+@command("fs", help="Act on File System data")
+class FsCmd(SqCommand):
+    def __init__(
+        self,
+        engine: str = "",
+        hostname: str = "",
+        start_time: str = "",
+        end_time: str = "",
+        view: str = "latest",
+        namespace: str = "",
+        format: str = "",
+        columns: str = "default",
+    ) -> None:
+        super().__init__(
+            engine=engine,
+            hostname=hostname,
+            start_time=start_time,
+            end_time=end_time,
+            view=view,
+            namespace=namespace,
+            columns=columns,
+            format=format,
+            sqobj=FsObj,
+        )
+
+    @command("show")
+    @argument("device", description="The Filesystem name")
+    def show(self, device: str = ''):
```

Comment:
Make device as list, not str. This is a bug on my part in how I did it in MacCmd.py. Do the same for mountPoint, but not for used-percent.

Its a list because we allow the user to specify more than one value